### DISCUSSION TITLE
在使用区域限制解锁脚本时视频下载功能无法获取到链接的问题

### DISCUSSION THREAD

**Discussion by CHAOS-mnt** [the1812/Bilibili-Evolved#3144] [提问 / Q&A]
海外党看番必须用[解锁脚本](https://greasyfork.org/zh-CN/scripts/25718-%E8%A7%A3%E9%99%A4b%E7%AB%99%E5%8C%BA%E5%9F%9F%E9%99%90%E5%88%B6)，但是视频下载插件似乎并不能获取到从反向代理地址获取的播放链接
![image](https://user-images.githubusercontent.com/22252583/159069635-f09e08b9-b8ef-4d23-acd5-f9539a04dcf1.png)
请问是否有可能兼容解锁脚本或者允许用户自定义api反代地址

**Comment by the1812** [comment]
你的网络必须是在浏览器中直连(不借助其他脚本)可观看时, 才可以下载, 其他区域限制情况一律不支持.
